packs/spell-effects/spell-effect-devil-form-levaloch.json

```diff
@@ -0,0 +1,154 @@
+                            "modifier": 23
+                        }
+                    },
+                    "speeds": {
+                        "climb": 30,
+                        "land": 30
+                    },
+                    "strikes": {
+                        "trident": {
+                            "ability": "str",
+                            "baseType": "trident",
+                            "category": "martial",
+                            "damage": {
+                                "damageType": "piercing",
+                                "dice": 2,
+                                "die": "d8",
+                                "modifier": 10
+                            },
+                            "group": "spear",
+                            "img": "systems/pf2e/icons/equipment/weapons/trident.webp",
+                            "label": "PF2E.Weapon.Base.trident",
+                            "modifier": 20,
+                            "traits": [
+                                "magical",
+                                "thrown-20",
+                                "unholy"
+                            ]
+                        }
+                    },
+                    "tempHP": 5,
+                    "traits": [
+                        "devil",
+                        "fiend"
+                    ],
+                    "weaknesses": [
+                        {
+                            "type": "holy",
+                            "value": 5
+                        }
+                    ]
+                }
+            },
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "label": "PF2E.BattleForm.DevilForm.LevalochSavesAgainstMoveLabel",
+                "option": "resist-move",
+                "toggleable": true
+            },
+            {
+                "key": "FlatModifier",
+                "label": "PF2E.BattleForm.DevilForm.LevalochFortitudeAgainstShoveLabel",
+                "predicate": [
+                    "action:shove",
+                    {
+                        "not": "resist-move"
+                    }
+                ],
+                "selector": "fortitude",
+                "type": "circumstance",
+                "value": 2
+            },
+            {
+                "key": "FlatModifier",
+                "label": "PF2E.BattleForm.DevilForm.LevalochSavesAgainstMoveLabel",
+                "predicate": [
+                    "resist-move"
+                ],
+                "selector": "saving-throw",
+                "type": "circumstance",
+                "value": 2
+            },
+            {
+                "category": "precision",
+                "diceNumber": 2,
+                "dieSize": "d6",
+                "hideIfDisabled": true,
+                "key": "DamageDice",
+                "predicate": [
+                    "item:range-increment:20",
```

Comment:
Why the range increment? Shouldn't it be for all melee strikes? Which would be just the melee strikes from the trident.

Reply:
I read it wrong - this was my last entry, and my eyes were already screaming by that point.
Fixing

Reply:
I hear that